### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by LesTR**

### CODE REVIEW

File: beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java [hunk 1]
```diff
@@ -100,12 +102,13 @@
     }
   }
 
+  @SuppressWarnings("unused")
```
Comment (je-ik) [inline beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java:105]: unused?

File: beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java [hunk 2]
```diff
@@ -100,12 +102,13 @@
...
   @Value
-  private final class BatchSnapshotDescriptor implements PCollectionDescriptor {
-    private final Pipeline pipeline;
-    private final DataAccessor dataAccessor;
-    private final long fromStamp;
-    private final long untilStamp;
+  private static class BatchSnapshotDescriptor implements PCollectionDescriptor {
```
Comment (je-ik) [inline beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java:107]: Do we want to remove the `final` modifier?

**Review by je-ik — COMMENTED** [review]
A few comments left.

File: beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java (hunk 1 above)
Comment (LesTR) [reply beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java]: Hmm, it was probably bug in Intellij, removed.

File: beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java (hunk 2 above)
Comment (LesTR) [reply beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java:104]: It's already `final` by @Value annotation from Lombok - https://projectlombok.org/features/Value

Comment (je-ik) [reply beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java:104]: ah, yep.

**Review by je-ik — APPROVED** [review]

**Pull request merged**